langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -215,6 +303,14 @@ public isolated function getContent(ProcessingInstruction|Comment x) returns str
 
 # Creates a new xml element item.
 #
+# ```ballerina
+# xml:createElement(
```

Comment:
Shall we add examples with and without each optional parameter

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/39130/commits/3a7088c41159f3c56a37443fa2036905f399eaa5